src/y0/examples.py

```diff
@@ -514,4 +515,19 @@ class Example:
     ],
 )
 
+# TODO: Add conditional independencies
+d_separation_example = Example(
+    name="D-separation example",
+    reference="http://web.mit.edu/jmn/www/6.034/d-separation.pdf",
+    graph=NxMixedGraph.from_edges(
```

Comment:
maybe we can make a field in the Example for example d-separations to test

Reply:
I don't think that's necessary: The expected conditional independencies are the things that are d-separated.  If we added the field, it would just cary duplicate information.  

In the long run, we might rename the field and related class, but for now I think the existing framework works.

Reply:
oh, thank you for the insight! Sorry that it's up to you guys to fill in the gaps in my knowledge 😁